cockroachcloud/egress-perimeter-controls.md

```diff
@@ -29,13 +29,11 @@ Further reading: [review how CockroachDB products differs in advanced security f
 Regardless of user-specific Egress Perimeter Control policy, egress is always permitted to services that are managed by Cockroach Labs and are essential to your cluster's functionality and ongoing operations.
 {{site.data.alerts.end}}
 
-## Prerequisites to using Egress Perimeter Controls with the Cloud Console API
+## Prerequisites
```

Comment:
FYI @MichaelTrestman we are changing the below wording in both the EPC and CMEK topics. I've also simplified the Prerequisites section heading in EPT.